e2-studio-star-rx72n-firmware/tests/test_rx_spi_link.c

```
@@ -0,0 +1,480 @@
+ * @return k_rx_ok on success
+ */
+static rx_err_t helper_init_link(bool fec_enabled)
+{
+    /* Initialize session state */
+    rx_err_t err = rx_session_init(&s_session);
+    if (err != k_rx_ok) {
+        return err;
+    }
+
+    /* Initialize SPI comm with session */
+    const rx_spi_comm_config_t spi_cfg = {
```

Clang (14.0.6)

[warning] variable 'spi_cfg' is not initialized

(cppcoreguidelines-init-variables)